src/core/machine-id-setup.c

```diff
@@ -165,7 +167,8 @@ int machine_id_setup(const char *root, sd_id128_t machine_id, sd_id128_t *ret) {
                 return log_error_errno(errno, "Failed to mount %s: %m", etc_machine_id);
         }
 
-        log_info("Installed transient %s file.", etc_machine_id);
+        if (!always_transient)
+                log_info("Installed transient %s file.", etc_machine_id);
```

Comment:
why not leave the log message in?

Reply:
My thinking was that it does not provide value in the case where it will always be installed transiently.  But if you prefer to have it log always, I'll change it.

Reply:
a compromise might be:

```c
log_full(always_transient ? LOG_DEBUG : LOG_INFO, "…
```